### ISSUE TITLE
Add some validator rules warnings about old tagging

### ISSUE THREAD

**Issue by Nakaner** [rurseekatze/OpenRailwayMap#206]
Add validator warnings for

* `etcs=yes/no` should be replaced by `railway:etcs=yes/no`
* `grade_of_track=-20/+20` should be removed
* `kursbuchstrecke=*` should be removed. The data is outdated and should be mapped using route relations.
* `lzb=yes/no` should be replaced by `railway:lzb=yes/no`
* `power_limit=*` is not surveyable and should be removed.
* `pzb=yes/no` should be replaced by `railway:pzb=yes/no`
* `radio=GSM-R` should be replaced by `railway:radio=gsm-r`
* `structure_gauge=*` should be replaced by ´railway:loading_gauge=*`
* ´traffic_mode=passenger` should be replaced by `railway:traffic_mode=passenger`
* ´traffic_mode=freight` should be replaced by `railway:traffic_mode=freight`
* ´traffic_mode=passenger;freight` should be replaced by `railway:traffic_mode=mixed` (Attention! Sometimes `railway:traffic_mode=*` has already been tagged)
* `track_class=*` should be replaced by `railway:track_class=*`
* `tilting_technology=yes/no` should be replaced by `railway:tilting_technologie`
* `operating_procedure=*` should be removed because it is not surveyable on the ground and should be stored in a separate database

**Comment by rurseekatze** [comment]
@Nakaner Fixed some markdown formatting issues in your comment.

**Comment by rurseekatze** [comment]
Some more old tagging:
* `catenary_mast=yes/no` at milestones should be replaced by `railway:milestone:catenary_mast=yes/no`
* `gauge=standard` should be replaced by `gauge=1435`

And some comments to @Nakaner's proposed retagging:
* Why should we remove `grade_of_track=-20/+20`? I would propose to replace it by `incline=*`.
* I would not remove `power_limit=*` (Oberstromgrenzwert, z.B. 600 A). It is not surveyable, but `ref=*` or `railway:ref=*` are not surveyable, too. The power limit could be important for routing applications. Although we have not any sources for this information (in Germany), I would keep this tag for cases of data imports.
* `tilting_technology=yes/no` should be replaced by the existing `railway:tilting=yes/no`, not `railway:tilting_technologie=*`
* `operating_procedure=*` is surveyable and so it should not be removed

**Comment by Nakaner** [comment]
`grade_of_track=+20/-20` ist not the incline its a strange encoded way to declare a railway line a "Steilstrecke" (line with high incline). https://wiki.openstreetmap.org/wiki/Proposed_features/detailed_Railway_Network#railtracks

**Comment by rurseekatze** [comment]
Yes, `grade_of_track=*` seems to be very strange. But I would suggest to ignore it instead of removing. I do not like to delete the work of other mappers without discussing it.
